### ISSUE TITLE
Cannot move scripts from a subfolder to the root level

### ISSUE THREAD

**Issue by AlCalzone** [ioBroker/ioBroker.javascript#469]
No matter where I drop it, it either lands in a subfolder or creates a fake folder under a script:
![grafik](https://user-images.githubusercontent.com/17641229/71688419-1e06c980-2da0-11ea-8ef5-99d5eb5f3b0c.png)
Here, "Anwesenheit" was moved.

**Comment by GermanBluefox** [comment]
please test with 4.3.6

**Comment by AlCalzone** [comment]
Still broken:
![ezgif com-video-to-gif(2)](https://user-images.githubusercontent.com/17641229/73675950-b18b2d00-46b3-11ea-8d41-e99bc2ec5d89.gif)

**Comment by AlCalzone** [comment]
And now I can't delete the script:
```
Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7Bcommand%7D&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
```

**Comment by AlCalzone** [comment]
I don't think this is fixed

**Comment by GermanBluefox** [comment]
Must be fixed in 4.10.1

**Comment by AlCalzone** [comment]
Indeed
